src/components/templates/Preplaya/Preplaya.tsx

```diff
@@ -20,18 +20,46 @@ import useLocationUpdateEffect, {
   updateLocationData,
 } from "utils/useLocationUpdateEffect";
 import { useUser } from "hooks/useUser";
-import { useParams } from "react-router-dom";
+import { useParams, useHistory } from "react-router-dom";
 import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
 import { faPlus, faMinus } from "@fortawesome/free-solid-svg-icons";
 import { throttle } from "lodash";
 
 import "./Preplaya.scss";
 import { peopleAttending } from "utils/venue";
+import { venueInsideUrl } from "utils/url";
 
 const isPlaced = (venue: Venue) => {
   return venue && venue.placement && venue.placement.x && venue.placement.y;
 };
 
+const getRandomInt = (max: number) => {
+  return Math.floor(Math.random() * Math.floor(max));
```

Comment:
the probability of getting max is zero. better to use `Math.round(Math.random..`